<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PApplicationUpdateWidget</class>
 <widget class="QWidget" name="PApplicationUpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Update Available</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="_WhatsNewLbl">
     <property name="text">
      <string>What's New</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWebEngineView" name="_WebEngine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="url">
      <url>
       <string>qrc:/updates.html</string>
      </url>
     </property>
     <property name="zoomFactor">
      <double>0.800000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_DownloadLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="_DownloadLbl">
       <property name="text">
        <string>Downloading: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="_Progress">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_ButtonLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="_BottomSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_DownloadBtn">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_InstallBtn">
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
